docs/index.md

```diff
@@ -53,15 +53,16 @@ On non-AMD platforms, like NVIDIA, HIP provides header files required to support
 
 :::{grid-item-card} Reference
 
-* {doc}`/doxygen/html/index`
+* [HIP runtime API](./reference/hip_runtime_api_reference)
+  * {ref}`modules_reference`
+  * {doc}`reference/hip_runtime_api/global_enum_and_defines`
+  * {doc}`reference/hip_runtime_api/data_structures`
+  * {doc}`reference/hip_runtime_api/files`
+* [HSA runtime API for ROCm](./reference/virtual_rocr)
```

Comment:
Why is this here? I don't see virtual_rocr in the file list, and is HSA from ROCR-runtime needed here?

Reply:
This is a separate issue.